src/App.tsx

```diff
@@ -15,36 +15,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SOTR_BY_ALPHABET: string = 'alphabet';
+const SOTR_BY_LENGTH: string = 'length';
+
+function SortGoodsFrom(
+  originArray: string[],
+  reverse: boolean,
+  sortBy: string,
```

Comment:
```suggestion
  isReversed: boolean,
  sortBy: sortType,
```
- Boolean variables should start from `is` or `has`
- `sortBy` isn't a string, it should have corresponding type